proto/viam/app/v1/end_user.proto

```diff
@@ -57,3 +60,17 @@ message UpdateAuthApplicationResponse {
   string application_id = 1;
   string application_name = 2;
 }
+
+message GetAuthApplicationRequest {
+  string org_id = 1;
+  string application_id = 2;
+}
+
+message GetAuthApplicationResponse {
+  string application_id = 1;
+  string application_name = 2;
+  string secret = 3;
```

Comment:
[minor] maybe we rename to `client_secret` to match FusionAuth's `application.oauthConfiguration.clientSecret` from their [API](https://fusionauth.io/docs/apis/applications)? also makes it a little easier to grep for this string.

Reply:
yeah makes sense to me. i can also rename the `secret` field in the `RegisterAuthApplication` request [here](https://github.com/viamrobotics/api/pull/506/files#diff-a9b71a46ce21ecb89e78164be49f980caadad4344c1116b519e8fc41dafd5d7eR47) to client_secret then?